Streamable/objects/obj_decklist_window/Create_0.gml

```diff
@@ -0,0 +1,49 @@
+/// @description
+event_inherited();
+
+var title = instance_create_layer(x, y, "UI", obj_title_bar, 
```

Comment:
this isn't a suggestion for this PR unless you _want_ to tackle it early, but there's a few windows now that have title, textbox, and two buttons that could be consolidated into one that takes in parameters via struct (title, initial text, reminder text, okay button text, okay/enter function to call with the text entered).

Reply:
Will take a look at this; probably a good idea